mp/src/materialsystem/stdshaders/pbr_dx9.cpp

```diff
@@ -359,6 +367,33 @@ BEGIN_VS_SHADER(PBR, "PBR shader")
             }
             pShaderAPI->SetPixelShaderConstant(PSREG_SELFILLUMTINT, color.Base());
 
+            if (g_pHardwareConfig->SupportsShaderModel_3_0())
+            {
+                // Setting up mrao scale
+                Vector mraoScale;
+                if (bHasMraoScale)
+                {
+                    params[info.mraoScale]->GetVecValue(mraoScale.Base(), 3);
+                }
+                else
+                {
+                    color = Vector{ 1.f, 1.f, 1.f };
+                }
+                pShaderAPI->SetPixelShaderConstant(41, mraoScale.Base());
+
+                // Setting up emission scale
+                Vector emissionScale;
+                if (bHasEmissionScale)
+                {
+                    params[info.emissionScale]->GetVecValue(emissionScale.Base(), 3);
+                }
+                else
+                {
+                    color = Vector{ 1.f, 1.f, 1.f };
```

Comment:
Should this be
```suggestion
                    emissionScale = Vector{ 1.f, 1.f, 1.f };
```
?

Reply:
I'll fix this first thing tomorrow if you haven't already by then

Reply:
Done in force push